keel/quotations/admin.py

```diff
@@ -16,7 +28,38 @@ class QuotationAdmin(admin.ModelAdmin):
 
 
 class QuotationMilestioneAdmin(CustomBaseModelAdmin):
-    pass
+    
+    def save_model(self, request, obj, form, change):
+        id = str(obj.pk)
```

Comment:
No need to assign obj.pk to id here, you could directly use str(obj.pk) in order_details dictionary.